src/Microsoft.Data.SqlClient/tests/UnitTests/Handlers/LoginChainTest.cs

```diff
@@ -38,17 +34,23 @@ public async void TestConnectivity()
             {
                 DataSource = _dataSource,
                 Encrypt = SqlConnectionEncryptOption.Mandatory,
-                TrustServerCertificate = true
+                TrustServerCertificate = true,
+                UserID = "sa",
+                Password = Environment.GetEnvironmentVariable("PASSWORD"),
```

Comment:
To ensure we can read configuration from config.json, I've opened https://github.com/dotnet/SqlClient/issues/2736, can be picked up by anyone in next iteration.